README.md

```diff
@@ -1,3 +1,42 @@
 # medic-builds-repo
 
 Design doc for builds repository.
+
+DDOCS are [pushed up by travis](https://github.com/medic/medic-webapp/blob/master/scripts/ci/push_to_staging.sh) with a specific name. These are then [pulled back down by api](https://github.com/medic/medic-api/blob/master/controllers/upgrade.js) and deployed by [horticulturalist](https://github.com/medic/horticulturalist).
+
+## Naming scheme
+
+Three valid examples: `medic:medic:3.0.0-beta.1`, `medic:medic:3.0.0`, `medic:medic:my-awesome-feature-branch`.
+
+The first two chunks between `:` is the namespace and the application name respectively. The namespace is just that: it has no logic relevant and is just used to scope things. The application name maps to the original ddoc name. It is what will be used to determine what to call the ddoc when deploying it.
```

Comment:
Even if the ddoc name is hardcoded in lots of places I would try to avoid hard coding it in any new code.  We might want to break up our ddoc(s) more or deploy them as separate apps one day in the future.  Since this is 3.0 code I thought I would mention it.